test/interface.js

```diff
@@ -69,5 +81,22 @@ describe('monitor-dog', function() {
         done();
       });
     });
+
+    describe('sockets-monitor', function () {
+      it('should expose a `start` method', function (done) {
+        var socketsMonitor = monitor.socketsMonitor;
+        expect(socketsMonitor.start).to.exist();
+        expect(typeof socketsMonitor.start).to.equal('function');
+        done();
+      });
+
+      it('should expose a `stop` method', function (done) {
+        var socketsMonitor = monitor.socketsMonitor;
+        expect(socketsMonitor.stop).to.exist();
+        expect(typeof socketsMonitor.stop).to.equal('function');
+        done();
+      });
+    });
+
```

Comment:
Are we actually testing that start and stop are making the appropriate gauge calls anywhere? If not, one thought is that you could setup a really fast monitor interval and just stop it once the first interval has fired (after checking to ensure the expected invocations of `gauge`).